ArduCopter/mode.h

```diff
@@ -438,9 +438,9 @@ class ModeAcro : public Mode {
     const char *name() const override { return "ACRO"; }
     const char *name4() const override { return "ACRO"; }
 
-    // get_pilot_desired_angle_rates - transform pilot's normalised roll pitch and yaw input into a desired lean angle rates
+    // get_pilot_desired_rates_cds - transform pilot's normalised roll pitch and yaw input into a desired lean angle rates
     // inputs are -1 to 1 and the function returns desired angle rates in centi-degrees-per-second
-    void get_pilot_desired_angle_rates(float roll_in, float pitch_in, float yaw_in, float &roll_out, float &pitch_out, float &yaw_out);
+    void get_pilot_desired_rates_cds(float roll_norm_in, float pitch_norm_in, float yaw_norm_in, float &roll_out_cds, float &pitch_out_cds, float &yaw_out_cds);
```

Comment:
Wondering about `norm` on the ends as a pseudo-unit

Reply:
I just moved it to the end.

Reply:
Looks like the arguments were renamed in the .cpp but the .h was missed.